backend/extensions/Squidex.Extensions/Actions/SignalR/SignalRActionHandler.cs

```diff
@@ -0,0 +1,104 @@
+                    .WithOptions(option =>
+                    {
+                        option.ConnectionString = key.ConnectionString;
+                        option.ServiceTransportType = ServiceTransportType.Transient;
+                    })
+                    .Build();
+                return serviceManager;
+            });
+        }
+
+        protected override async Task<(string Description, SignalRJob Data)> CreateJobAsync(EnrichedEvent @event, SignalRAction action)
+        {
+            var hubName = await FormatAsync(action.HubName, @event);
+
+            string requestBody;
+
+            if (!string.IsNullOrEmpty(action.Payload))
+            {
+                requestBody = await FormatAsync(action.Payload, @event);
+            }
+            else
+            {
+                requestBody = ToEnvelopeJson(@event);
+            }
+
+            var ruleDescription = $"Send SignalRJob to signalR hub '{hubName}'";
+
+            var ruleJob = new SignalRJob
+            {
+                ConnectionString = action.ConnectionString,
+                HubName = hubName,
+                Action = action.ActionType,
+                MethodName = action.MethodName,
+                Group = await FormatAsync(action.Group, @event),
+                User = await FormatAsync(action.User, @event),
+                Payload = requestBody
+            };
+            return (ruleDescription, ruleJob);
+        }
+
+        protected override async Task<Result> ExecuteJobAsync(SignalRJob job, CancellationToken ct = default)
+        {
+            var signalR = await clients.GetClientAsync((job.ConnectionString, job.HubName));
+            var signalRContext = await signalR.CreateHubContextAsync(job.HubName);
+
+            var methodeName = !string.IsNullOrWhiteSpace(job.MethodName) ? job.MethodName : "push";
+
+            switch (job.Action)
+            {
+                case ActionTypeEnum.BROADCAST:
+                    await signalRContext.Clients.All.SendAsync(methodeName, job.Payload);
+                    break;
+                case ActionTypeEnum.USER:
+                    await signalRContext.Clients.User(job.User).SendAsync(methodeName, job.Payload);
+                    break;
+                case ActionTypeEnum.USERS:
+                    var userIds = job.User.Split('\n');
+                    await signalRContext.Clients.Users(userIds).SendAsync(methodeName, job.Payload);
+                    break;
+                case ActionTypeEnum.GROUP:
+                    await signalRContext.Clients.Group(job.Group).SendAsync(methodeName, job.Payload);
+                    break;
+                case ActionTypeEnum.GROUPS:
+                    var groupIds = job.Group.Split('\n');
+                    await signalRContext.Clients.Groups(groupIds).SendAsync(methodeName, job.Payload);
+                    break;
+            }
+
+            await signalRContext.DisposeAsync();
```

Comment:
Please use ´await using (...)` here

Reply:
ok, delete DisposeAsync and use 'await using(...)'